ProjectToTest/User.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Reflection;
```

Comment:
Why we need System.Reflection, again??? It is hard to me to understand, well first I do not know what this class can do, but word 'reflection', looks to me without connection to the tasks of the excercise,